pages.id/common/gh.md

```diff
@@ -0,0 +1,37 @@
+# gh
+
+> Memudahkan pengaksesan GitHub dari command-line.
+> Beberapa subcommands seperti `gh config` memiliki dokumentasi sendiri.
+> Informasi lebih lanjut: <https://cli.github.com/>.
+
+- Mengklon sebuah GitHub repositori di lokal
+
+`gh repo clone {{pemilik}}/{{repositori}}`
+
+- Membuat isu baru:
+
+`gh issue create`
+
+- Melihat dan filter issue yang sedang open pada repositori:
+
+`gh issue list`
+
+- Melihat isu di browser:
+
+`gh issue view --web {{nomor_issue}}`
```

Comment:
```suggestion
`gh issue view --web {{nomor_isu}}`
```
Is this translation also applicable here?

Reply:
on it